### ISSUE TITLE
Warnings in using Knet in a new module

### ISSUE THREAD

**Issue by ngphuoc** [denizyuret/Knet.jl#52]
I defined module "abc.jl" as follows:

```julia
__precompile__()
module abc
export f
using Knet

function f()
  x = 1
end

end
```
Knet produced the following warning:

```
julia> using abc
INFO: Recompiling stale cache file /home/phuoc/.julia/lib/v0.5/abc.ji for module abc.
WARNING: eval from module Knet to abc:    
Expr(:||, Expr(:call, :==, Expr(:ccall, Expr(:tuple, "nvmlInit", "libnvidia-ml")::Any, :UInt32, Expr(:tuple)::Any)::Any, 0)::Any, Expr(:call, :error)::Any)::Any
  ** incremental compilation may be broken for this module **

WARNING: eval from module Knet to abc:    
Expr(:||, Expr(:call, :==, Expr(:ccall, Expr(:tuple, "nvmlDeviceGetCount", "libnvidia-ml")::Any, :UInt32, Expr(:tuple, Expr(:curly, :Ptr, :Cuint)::Any)::Any, Array{UInt32, 1}[0x00000000])::Any, 0)::Any, Expr(:call, :error)::Any)::Any
  ** incremental compilation may be broken for this module **

INFO: Knet using GPU 1
INFO: Knet using GPU 1
```

**Comment by ekyurdakul** [comment]
It should work if you remove `__precompile__()`.

**Comment by denizyuret** [comment]
Fixed in version 395c970 2017-02-11.
